servers/src/common/adapters.rs

```diff
@@ -477,6 +482,66 @@ where
 	fn get_tmpfile_pathname(&self, tmpfile_name: String) -> PathBuf {
 		self.chain().get_tmpfile_pathname(tmpfile_name)
 	}
+
+	fn get_kernel_segment(
+		&self,
+		hash: Hash,
+		id: SegmentIdentifier,
+	) -> Result<Segment<TxKernel>, chain::Error> {
+		if id.height < 9 || id.height > 13 {
```

Comment:
We should pull these min and max heights out into `const` so its easier to see them all together.

Reply:
Done